src/sdk/search/useSearchHistory.ts

```diff
@@ -5,9 +5,13 @@ const storageKey = 'main::store::searchHistory'
 const MAX_HISTORY_SIZE = 5
 
 export default function useSearchHistory(
+  history: string[] = [],
```

Comment:
Can you explain why did you add this?

Reply:
I did this so we could show the `history` items in the `SearchHistory` component inside Storybook. 

This component uses `useStorage` to fetch this data, but inside Storybook there would be no way for us to populate this and the component being rendered with the items. 

This way, for the Storybook example, we can pass this initial data but it defaults to an empty array when not passed, which was already [the previous value](https://github.com/vtex-sites/gatsby.store/pull/7/files?file-filters%5B%5D=.ts&show-deleted-files=true&show-viewed-files=true#diff-912012eb2e9f77d2bb5c5c905a59393b2d844675034d4b61fc2e6959b529b885L10).
https://github.com/vtex-sites/gatsby.store/blob/e14452f08436a34dcb39e49412d5d8673943c09a/src/components/search/History/History.stories.tsx#L14